src/create-rpc-server.ts

```diff
@@ -31,9 +31,23 @@ const resources: Record<string, Record<string, Parameters<typeof createServer>[1
     GET: (req, res) => {
       // force nodejs to generate a heapdump
       // you can analyze the heapdump with https://github.com/facebook/memlab#heap-analysis-and-investigation to get some really useful insights
-      const filename = writeHeapSnapshot(`./snapshot-dir/${(new Date()).toISOString()}.heapsnapshot`)
       res.writeHead(200, { 'Content-Type': 'text/plain' })
```

Comment:
```suggestion
      res.writeHead(200, { 'Content-Type': 'application/json' })
```

Reply:
@2color like this?

Reply:
Perhaps also add ``Content-Disposition: attachment; filename=`${(new Date()).toISOString()}.heapsnapshot` ``

Reply:
Please suggest an edit 🙏